src/Application/Projects/Commands/DeleteProject/DeleteProjectCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CoduTeam.Application.Common.Interfaces;
+
+namespace CoduTeam.Application.Projects.Commands.DeleteProject;
+
+public record DeleteProjectCommand(int Id) : IRequest;
+
+public class DeleteProjectCommandHandler(IApplicationDbContext dbContext) : IRequestHandler<DeleteProjectCommand>
```

Comment:
Also need to add validator that will check that current user is owner of project